edscrapers/tools/dashboard/pages/trends.py

```diff
@@ -133,6 +161,11 @@ def all_domain_quality_series():
         dfs_oela = pd.concat(dfs_oela, ignore_index=True)
         concat_lst.append(dfs_oela)
 
+    dfs_oese = get_series('oese')
+    if dfs_oese:
+        dfs_oese = pd.concat(dfs_oese, ignore_index=True)
+        concat_lst.append(dfs_oese)
+
```

Comment:
can you keep it a bit more DRY? Something like 

```python
for office in ['oese', 'rems']:
    dfs = get_series(office)
    if dfs:
        dfs = pd.concat(dfs, ignore_index=True)
        concat_lst.append(dfs)
```

Reply:
Sure. Nice idea.

Reply:
Done